### PR TITLE
add changelog files

### PR DESCRIPTION
**Pull request by stentrop**
### All changes meet the following requirements
- [ ] Changelog entry was added
- [ ] Changes have been documented
- [ ] Changes have been tested by the author
- [ ] Changes have been tested by the reviewer

@plentymarkets/plentyshop

### CODE REVIEW

File: meta/documents/changelog_de.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
 # Release Notes für plentyShop LTS
 
+## v5.0.41 (2021-10-20) <a href="https://github.com/plentymarkets/plugin-ceres/compare/5.0.40...5.0.41" target="_blank" rel="noopener"><b>Übersicht aller Änderungen</b></a>
+
+### Hinzugefügt
+
+* Ein neuer LayoutContainer "Ceres::PageDesign.AfterOpeningHeadTag" wurde hinzugefügt. @FelixRies
```
Comment (ksted) [inline meta/documents/changelog_de.md]:
```suggestion
* Der LayoutContainer "Ceres::PageDesign.AfterOpeningHeadTag" wurde hinzugefügt. @FelixRies
```

File: meta/documents/changelog_de.md [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+
+### Geändert
+
+* Rebranding **Ceres** wird jetzt unter **plentyShop LTS** beworben.
```
Comment (ksted) [inline meta/documents/changelog_de.md]:
```suggestion
* Rebranding: **Ceres** wird jetzt unter **plentyShop LTS** beworben.
```

File: meta/documents/changelog_de.md [hunk 3]
```diff
@@ -1,5 +1,30 @@
...
+
+### Behoben
+
+* Im Standard Login Template wurde der Input des Gastlogin-Modals nach dem öffnen nicht fokussiert. Dies wurde behoben.
```
Comment (ksted) [inline meta/documents/changelog_de.md]:
```suggestion
* Im Standard Login-Template wurde der Input des Gastlogin-Modals nach dem Öffnen nicht fokussiert. Dies wurde behoben.
```

File: meta/documents/changelog_de.md [hunk 4]
```diff
@@ -1,5 +1,30 @@
...
+* Im Assistenten wurde nicht dargestellt, das die Bestelleigenschaften aktiviert wurden. Das wurde behoben.
```
Comment (ksted) [inline meta/documents/changelog_de.md]:
```suggestion
* Im Assistenten wurde nicht dargestellt, wenn die Bestelleigenschaften aktiviert wurden. Das wurde behoben.
```

File: meta/documents/changelog_de.md [hunk 5]
```diff
@@ -1,5 +1,30 @@
...
+
+### Angepasste Templates
+
+- Im Zuge des Releases von Ceres 5.0.41 gab es Änderungen an einer Template-Datei, die für Theme-Entwickler relevant ist. Die Verlinkung führt direkt zu der umgesetzten Änderung in der entsprechenden Datei.
+- [resources/views/PageDesign/PageDesign.twig](https://github.com/plentymarkets/plugin-ceres/pull/3051/files#diff-6103acda40fc331c6f5ba92faab9976e92848d8fde0e9af7fe34abd593885128)
+
+### Changed templates
+
+- In Ceres 5.0.41 we made a change to a template file which is relevant for theme developers. You can find the changed template below. The link directs you to the effected changes in the corresponding file.
```
Comment (ksted) [inline meta/documents/changelog_de.md]: Falsche Datei?

File: meta/documents/changelog_en.md [hunk 6]
```diff
@@ -1,5 +1,25 @@
 # Release Notes for plentyShop LTS
 
+## v5.0.41 (2021-10-20) <a href="https://github.com/plentymarkets/plugin-ceres/compare/5.0.40...5.0.41" target="_blank" rel="noopener"><b>Overview of all changes</b></a>
+
+### Added
+
++ A new LayoutContainer "Ceres::PageDesign.AfterOpeningHeadTag" was added. @FelixRies
```
Comment (ksted) [inline meta/documents/changelog_en.md]:
```suggestion
+ The LayoutContainer "Ceres::PageDesign.AfterOpeningHeadTag" was added. @FelixRies
```

File: meta/documents/changelog_en.md [hunk 7]
```diff
@@ -1,5 +1,25 @@
...
+
+### Changed
+
+* Rebranding **Ceres** is now promoted as **plentyShop LTS**.
```
Comment (ksted) [inline meta/documents/changelog_en.md]:
```suggestion
* Rebranding: **Ceres** is now promoted as **plentyShop LTS**.
```

File: meta/documents/changelog_en.md [hunk 8]
```diff
@@ -1,5 +1,25 @@
...
+
+### Fixed
+
+* In the standard login template, the input of the guest login modal was not focused after opening. This has been fixed.
+* In the assistant it was not shown that the order properties were activated. This has been fixed.
```
Comment (ksted) [inline meta/documents/changelog_en.md]:
```suggestion
* The assistant didn't show when the order properties were activated. This has been fixed.
```

**Review by felixgehrmann — APPROVED** [review]

**Review by Dominik2809 — APPROVED** [review]

**Pull request merged**